recipes/glib/all/conanfile.py

```diff
@@ -198,7 +198,10 @@ def package_info(self):
                 self.cpp_info.components["gio-2.0"].requires.append("libmount::libmount")
             if self.options.with_selinux:
                 self.cpp_info.components["gio-2.0"].requires.append("libselinux::libselinux")
-        if self.settings.os != "Windows":
+        if self.settings.os == "Windows":
+            self.cpp_info.components["gio-windows-2.0"].requires = ["gobject-2.0", "gmodule-2.0", "gio-2.0"]
```

Comment:
`gmodule-no-export-2.0` instead of `gmodule-2.0`?
https://gitlab.gnome.org/GNOME/glib/-/blob/master/gio/meson.build#L841

Reply:
the difference between these two is the  `-Wl,--export-dynamic` which I don't know how to handle with conan

Reply:
scratch my previous comment, I forgot that I already defined these no-export components.
on Windows the foo and foo-no-export are actually identical anyway, because 
[`export_dynamic_ldflags` is empty](https://gitlab.gnome.org/GNOME/glib/-/blob/master/meson.build#L2219)